src/components/TableRamais/Table.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="q-pa-xl box-shadow mt-10 mx-8">
+    <div class="row justify-between q-pa-md">
+      <div>
+        <q-btn class="bg-green text-white ml-12" label="Adicionar Ramal" />
+      </div>
+      <q-input label="Buscar Ramal" color="indigo-8">
+        <template v-slot:append>
+          <q-icon name="search" color="indigo-8" />
+        </template>
+      </q-input>
+    </div>
+    <q-markup-table class="q-px-xl meu-tabela no-box-shadow">
+      <thead>
+        <tr class="text-indigo-8">
```

Comment:
pode criar um constante com os nomes das colunas e faz um v-for pra cada th, mas isso pode ficar para um proximo pr